keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -252,24 +262,26 @@ status:
 
 #### BucketClass
 
-A cluster-scoped custom resource. The bucket class defines a release policy, and specifies driver specific parameters, such as region, bucket lifecycle policies, etc., as well as the name of the driver.  The driver name is used to filter BuckeContenst meant to be handled by a given driver.  In static bucket workflows, the driver name may be empty if the object bucket is defined.
+A cluster-scoped custom resource. The BucketClass defines a release policy, and specifies driver specific parameters, such as region, bucket lifecycle policies, etc., as well as the name of the driver.  The driver name is used to filter BuckeContenst meant to be handled by a given driver.  In static bucket workflows, the driver name may be empty if the object bucket is defined.
 
 ```yaml
 apiVersion: cosi.io/v1alpha1
 kind: BucketClass
 metadata:
   name: 
 provisioner: [1]
-config: string:string [2]
-releasePolicy: {"Delete", "Retain"} [3]
-targetBucket: [5]
-secretName: [6]
+supportedProtocols: [2]
+accessMode: {"ro", "rw"} [3]
```

Comment:
plural (like in description below)

Reply:
The description doesn't make the distinction that it should be plural.  I doesn't really make sense to allow both at the same time anyway.